engineering/README.md

```diff
@@ -0,0 +1,155 @@
+
+## Planning
+[ ] Before Friday Engineering meeting: Each team member collects notes on their top priorities for the coming week, progress on existing items, concerns etc. and adds bullet list items to Friday engineering meeting agenda.
+[ ] Friday Engineering meeting: team members discuss their priorities and concerns along with those of the community.
+[ ] Meeting deliverable: consensus on a few bullet point priorities for the following week that will be presented to co-founders. These should be listed at the top of the agenda document and link to github issues if they exist.
+[ ] Co-founders & team members consider these proposed engineering priorities before the weekly team meeting.
+[ ] Co-founders present their priorities for the upcoming week having taken into account the engineering teams priorities & team discusses.
+
+## Meetings
+
+Meetings are expensive, especially for creative people like designers and engineers. A 15 minute meeting can mean an hour or more of lost productivity. Meetings are also synchronous and mean that only those that can be available at that certain time have a chance to give input into the discussion. As such, we prefer to default to conducting discussions on github issues or in the forum and to minimize meetings whenever possible.
+
+### Meeting rules
+
+[ ] Meetings always begin on time: starting late penalizes the punctual and rewards the tardy. You should connect to the meeting conference call or arrive in the conference room a few minutes before the start time.
+[ ] Meetings always end on time: our team member and community member time is very valuable and they have other commitments. Disciplined meetings can cover the required topics within the allocated time.
+[ ] No meeting shall proceed without a written agenda: if it's not important enough to create an agenda, it's not important enough to hold a meeting
+[ ] Each meeting should have a stated goal, deliverable or outcome. This helps keep the meeting on track.
+
+**Calling a meeting**
+*Responsible person:* Person who called the meeting (or delegate)
+
+[ ] Create an agenda and notes in a paper document
+[ ] Create a zoom meeting link
+[ ] Confirm attendee availability
+[ ] Create a forum post in the meetings category with date, time, agenda & zoom meeting link*
+[ ] Send a calendar invite that includes the zoom meeting link & link to agenda
+
+** Public meetings only*
+
+### Agenda contents
+*Responsible person:* Person who called the meeting (or delegate)
+
+[ ] Items to be dicussed
+[ ] Desired outcome or deliverable
+[ ] Links to any relevant previous meeting notes
+[ ] Links to any github issues or other items that provide background info so that attendees can be up to speed
+
+**Holding meetings**
+*Responsible person:* Person who called the meeting (or delegate)
+
+[ ] At meeting start, paste link to forum meeting post in slack #engineering channel*
+[ ] Nominate a person to take notes
```

Comment:
Should we say also that the notes will always go to the forum post?

Reply:
it says that later down in the checklist